### PR TITLE
Added data model and data dictionary to srs

### PR DESCRIPTION
**Pull request by akshitg3**
#60
* Added data model and data dictionary to srs

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -430,9 +430,113 @@ \subsection{Specifying a Business Use Case (BUC)}
...
+      \item \textbf{Relationships:}
+      \begin{itemize}
+        \item An AdminUser can view other users data (many-to-many relationship)
+        \item An AdminUser can edit multiple documents (many-to-many relationship) 
+      \end{itemize}
+    \end{itemize}
+
+    \item GeneralUser
+    \begin{itemize}
+      \item This class is a sub-class of User class which reprsents a general user of the poral (such as a someone from the city staff or a contractor)
```
Comment (Spitgranger) [inline docs/SRS-Volere/SRS.tex]: This line is over 80 characters

**Comment by kyle4908** [comment]
I'll fix the formatting

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -430,9 +430,113 @@ \subsection{Specifying a Business Use Case (BUC)}
...
+    \end{itemize}
+
+
+    \item Document
+    \begin{itemize}
+      \item This class represents a document stored on the portal
+      \item \textbf{Attributes:}
+      \begin{itemize}
+        \item id: int -- The id of the document
+        \item category: String -- The category of the document (such as "health and safety")
```
Comment (Spitgranger) [inline docs/SRS-Volere/SRS.tex]: These show up as syntax errors use `` for quotation marks

File: docs/SRS-Volere/SRS.tex [hunk 3]
```diff
@@ -430,9 +430,128 @@ \subsection{Specifying a Business Use Case (BUC)}
...
 \subsection{Data Dictionary}
-\lips
+\textbf{Classes}
+\begin{itemize}
+  \item  User
+    \begin{itemize}
+      \item This class represents a user of the portal
+      \item \textbf{Attributes:}
+        \begin{itemize}
+          \item userData: UserData -- It is the data related to the
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:446]: we got feedback about this on the first deliverable didn't we "Use Unicode characters. For example, you can use right dash (just the minus sign `-`, endash `–` or emdash `—`) instead using multiple `-`s."

Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex:446]: oh wait actually I think this is fine idk what that line of feedback meant which is why I commented, but I think this is fine

**Review by kyle4908 — APPROVED** [review]

**Pull request merged**